src/static_canvas.class.ts

```diff
@@ -257,6 +261,28 @@ export class StaticCanvas extends createCollectionMixin(
    */
   disposed?: boolean;
 
+  renderAndResetBound: () => void;
+  requestRenderAllBound: () => StaticCanvas;
+
+  // TODO: move to canvas
+  interactive: boolean;
+  upperCanvasEl: HTMLCanvasElement;
+  contextTop: CanvasRenderingContext2D;
+  wrapperEl: HTMLDivElement;
+  cacheCanvasEl: HTMLCanvasElement;
+  protected _isCurrentlyDrawing: boolean;
+  freeDrawingBrush: BaseBrush;
+  _activeObject: FabricObject;
```

Comment:
All of there belong to canvas along with `drawControls` and `_applyCanvasStyle`

Reply:
So how this passed before withtout the ts-nocheck on top?
How did i get a clea build without those types?